frontend/e2e/application-noncommercial-group.e2e-spec.ts

```diff
@@ -39,110 +39,93 @@ describe('Apply for a noncommercial group use permit', () => {
   beforeEach(() => {
     page = new NoncommercialGroupForm();
   });
+
```

Comment:
We submit the applications in the tests and the application will return an error and fail the test if communication with the server fails. If successful, the success page is tested